initParam.m

```diff
@@ -25,18 +25,27 @@
 Param.numSubFramesUE = 25;% Integer used to set the number of RBs for the uplink
 Param.numMacro = 1;% Integer used to specify the number of macro eNodeBs in the scenario (currently only 1)
 Param.numMicro = 1;% Integer used to specify the number of micro eNodeBs in the scenario
-Param.microPos = 'uniform';% Array of char to deicde the positioning of the micro BS
+Param.microPos = 'clusterized';% Array of char to deicde the positioning of the micro BS
```

Comment:
Could you add a comment of the possible positioning options? e.g. 'uniform', 'random' and 'clusterized'